x/bank/internal/types/expected_keepers.go

```diff
@@ -0,0 +1,18 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/exported"
+)
+
+// AccountKeeper defines the account contract that must be fulfilled when
+// creating a x/bank keeper.
+type AccountKeeper interface {
```

Comment:
Couldn't this Keeper interface be part of `x/auth/exported` as well? Or do we want this contract to be explicitly in `x/bank`?

Reply:
This defines only the expected Account Keeper functions that the Bank Keeper will need. Other modules might use a different subset of the original AccountKeeper functions that they will require